src/SourceGenerators/Uno.UI.SourceGenerators.Tests/XamlCodeGeneratorTests/Out/WNBOP/XamlCodeGenerator_GlobalStaticResources.cs

```diff
@@ -0,0 +1,55 @@
+﻿// <autogenerated />
+namespace MyProject
+{
+	/// <summary>
+	/// Contains all the static resources defined for the application
+	/// </summary>
+	public sealed partial class GlobalStaticResources
+	{
+		static bool _initialized;
+		private static bool _stylesRegistered;
+		private static bool _dictionariesRegistered;
+		internal static global::Uno.UI.Xaml.XamlParseContext __ParseContext_ { get; } = new global::Uno.UI.Xaml.XamlParseContext()
+		{
+			AssemblyName = "TestProject",
+		}
+		;
```

Comment:
nit:

Reply:
This is generated code, it will stay like this :)